tests/core/contracts/conftest.py

```diff
@@ -387,51 +387,55 @@ def ArraysContract(web3, ARRAYS_CONTRACT):
 
-CONTRACT_FALLBACK_FUNCTION_RUNTIME = "606060405260043610603f576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff1680633bc5de30146053575b3415604957600080fd5b6001600081905550005b3415605d57600080fd5b60636079565b6040518082815260200191505060405180910390f35b600080549050905600a165627a7a72305820045439389e4742569ec078687e6a0c81997709778a0097adbe07ccfd9f7b1a330029"  # noqa: E501
+CONTRACT_PAYABLE_TESTER_RUNTIME = "6080604052600436106053576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff1680634c3e7178146058578063c6803622146060578063e4cb8f5c14608c575b600080fd5b605e60a0565b005b348015606b57600080fd5b50607260bc565b604051808215151515815260200191505060405180910390f35b348015609757600080fd5b50609e60ce565b005b60016000806101000a81548160ff021916908315150217905550565b6000809054906101000a900460ff1681565b60016000806101000a81548160ff0219169083151502179055505600a165627a7a723058201c5cb2fb1a9777ae66d80269e9bba04e786e5657236850bd543d4d0e7e31ecde0029"  # noqa: E501
 
-CONTRACT_FALLBACK_FUNCTION_ABI = json.loads('[{"constant": false, "inputs": [], "name": "getData", "outputs": [{"name": "r", "type": "uint256"}], "payable": false, "stateMutability": "nonpayable", "type": "function"}, {"inputs": [], "payable": true, "stateMutability": "payable", "type": "constructor"}, {"payable": false, "stateMutability": "nonpayable", "type": "fallback"}]')  # noqa: E501
+CONTRACT_PAYABLE_TESTER_ABI = json.loads('[{"constant": false, "inputs": [], "name": "doPayableCall", "outputs": [], "payable": true, "stateMutability": "payable", "type": "function"}, {"constant": true, "inputs": [], "name": "wasCalled", "outputs": [{"name": "", "type": "bool"}], "payable": false, "stateMutability": "view", "type": "function"}, {"constant": false, "inputs": [], "name": "doNoValueCall", "outputs": [], "payable": false, "stateMutability": "nonpayable", "type": "function"}]')  # noqa: E501
 
 
 @pytest.fixture()
-def FALLBACK_FUNCTION_CODE():
-    return CONTRACT_FALLBACK_FUNCTION_CODE
+def PAYABLE_TESTER_CODE():
+    return CONTRACT_PAYABLE_TESTER_CODE
 
 
 @pytest.fixture()
-def FALLBACK_FUNCTION_RUNTIME():
-    return CONTRACT_FALLBACK_FUNCTION_RUNTIME
+def PAYABLE_TESTER_RUNTIME():
+    return CONTRACT_PAYABLE_TESTER_RUNTIME
 
 
 @pytest.fixture()
-def FALLBACK_FUNCTION_ABI():
-    return CONTRACT_FALLBACK_FUNCTION_ABI
+def PAYABLE_TESTER_ABI():
+    return CONTRACT_PAYABLE_TESTER_ABI
 
 
 @pytest.fixture()
-def FALLBACK_FUNCTION_CONTRACT(FALLBACK_FUNCTION_CODE,
-                               FALLBACK_FUNCTION_RUNTIME,
-                               FALLBACK_FUNCTION_ABI):
+def PAYABLE_TESTER_CONTRACT(PAYABLE_TESTER_CODE,
```

Comment:
cc @njgheorghita I was looking at this stuff and it might be a good test-bed for seeing how py-ethpm and maybe `pytest-ethereum` could clean up these test fixtures.